### ISSUE TITLE
Google search result link gives 'Unable to retrieve content' error

### ISSUE THREAD

**Issue by ntrel** [oxyry/reabble#52]
I thought Reabble worked with search result links, but recently it seems to have stopped working. This link gives an 'Unable to retrieve content' red banner when I click 'Load':

https://www.google.com/url?sa=t&source=web&rct=j&opi=89978449&url=https://www.ombudsman.org.uk/complaints-womens-state-pension-age&ved=2ahUKEwjT1KfRjrGKAxXpSkEAHdkpJZYQFnoECCwQAQ&usg=AOvVaw0b7We1PzWYvMrJqhPBNWnH

If I strip the google prefix and suffix arguments, this works:
https://www.ombudsman.org.uk/complaints-womens-state-pension-age

Please could you fix/implement this automatically?

**Comment by weijarz** [comment]
A new feature has been added to try another URL embedded in the URL parameter. You can retry if it solves the problem.

**Comment by ntrel** [comment]
Thanks, but I refreshed the page and tried the google link above, it didn't work.

**Comment by weijarz** [comment]
Previous results will be cached for a few days, try another link.

**Comment by ntrel** [comment]
I've encountered this bug multiple times since, still doesn't work unfortunately.

**Comment by weijarz** [comment]
Google encrypts their links using a new, more sophisticated way, so it's hard to get the original article address.

**Comment by ntrel** [comment]
No, for me I only need to strip the start and end of if the URL: 

https://www.google.com/url?sa=t&source=web&rct=j&opi=89978449&url=https://www.wired.com/story/best-apple-tv-plus-shows/&ved=2ahUKEwi0vpGTy_2LAxVlT0EAHTc_JgAQFnoECGsQAQ&usg=AOvVaw1D_7dUn81lthTCLekJ15Ry

Above is the Google result link, trimmed: 

https://www.wired.com/story/best-apple-tv-plus-shows

**Comment by weijarz** [comment]
Okay, now it's correctly handling the URL address you're talking about.
